src/Core/Settings.cpp

```diff
@@ -40,10 +38,15 @@ namespace ErrorCodes
   * Note: as an alternative, we could implement settings to be completely dynamic in the form of the map: String -> Field,
   *  but we are not going to do it, because settings are used everywhere as static struct fields.
   *
-  * `flags` can be either 0 or IMPORTANT.
+  * `flags` can be either 0 or IMPORTANT + a Tier (PRODUCTION | BETA | EXPERIMENTAL)
```

Comment:
It was not obvious to me, that 0 = PRODUCTION, until I checked declaration. Maybe state it explicitly and/or change 0 to PRODUCTION, wdyt?

Reply:
Agree, I'll improve the comment